pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/JdbiCompositeQuestion.java

```diff
@@ -83,13 +83,14 @@ void insertChild(@Bind("parentQuestionId") long parentQuestionId, @Bind("childQu
         return findParentQuestionIdByChildQuestionId(questionDto.getId());
     }
 
+    // NOT USED
```

Comment:
I haven't yet cleaned up these `Jdbi*` question interfaces and cache classes. I'll do that in the next PR.